pkg/agent/nodeportlocal/k8s/npl_controller.go

```diff
@@ -64,6 +64,10 @@ type NPLController struct {
 	podIPLock   sync.RWMutex
 }
 
+var (
+	SyncRuleInterval = 60 * time.Second
```

Comment:
Better to make this configuration static and private. As it is only used in tests. And I think as the integration tests do not call `go wait.Until()` to sync the rules, the configuration seems to be meaningless.